ui/src/components/topology/wp-data.js

```diff
@@ -19,7 +19,7 @@ module.exports = [{
     "be227788-74f1-4e5b-a85f-b5c71cbae8d8"
   ],
   "healthy": true,
-  "datacentres": 1
+  "datacenter": 1
```

Comment:
@geek that property wont matter be used in the cp portal. It existed in the context of "in how many dc's is this service running on".